smrf/tests/test_lakes_wind_ninja.py

```diff
@@ -21,7 +21,10 @@ def setUpClass(cls):
         """
         super().setUpClass()
 
-        run_smrf(cls.config_file)
+        cls.smrf = run_smrf(cls.config_file)
+
+    def setUp(self):
+        self.assert_thread_variables()
```

Comment:
Is this to ensure an empty start?
Maybe not check for every test case?

Reply:
I was having trouble with it in the `setupClass` so I moved it here. It's not going to change from test to test.